packages/icon/src/template.ts

```diff
@@ -73,6 +74,11 @@ module.exports = function template(
         color: \${fill};
       \`;
 
+      const noFlexSizing = css\`
```

Comment:
Why are we setting flex-grow to 0? IIRC, don't we find ourselves adding that property to maintain the icon's size

Reply:
You're right, this should be the implicit value already. By being explicit here, I hoped to make it easier to debug issues around this related to setting that value to something other than `0`.

If you feel strongly, I can remove this.

Reply:
I thought we had discussions around setting this to 1 explicitly rather than setting to 0 explicitly.

Reply:
Fair point, I'll remove.